src/main/java/core/basesyntax/io/write/impl/ReportWriterImpl.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax.io.write.impl;
+
+import core.basesyntax.io.write.ReportWriter;
+
+public class ReportWriterImpl implements ReportWriter {
+    private final ReportWriter writer;
+
+    public ReportWriterImpl(ReportWriter writer) {
+        this.writer = writer;
+    }
+
+    @Override
+    public void write(String report, String reportPath) {
+        writer.write(report, reportPath);
+    }
+}
```

Comment:
The ReportWriterImpl class is currently just delegating its write method to another ReportWriter instance without adding any extra logic or value. This is a violation of the SOLID principles, specifically the 'Single Responsibility Principle'. Consider removing this class if it's not adding any extra functionality, or implement the write logic directly in this class.